core/cmd/app.go

```diff
@@ -627,6 +626,12 @@ func NewApp(client *Client) *cli.App {
 							Action: client.PrepareTestDatabase,
 							Flags:  []cli.Flag{},
 						},
+						{
```

Comment:
We should add a "version" command here also which shows the latest version

Reply:
Do you mean this version? https://github.com/smartcontractkit/chainlink/blob/7af51128ef9f9e4289a2142564d0aee0420cb120/core/store/orm/orm.go#L170

It returns `random_<some long number>`

Reply:
i think you can get the last db migration version via `getLastRunMigration` in migration_helper.go